client/src/views/Projects.vue

```diff
@@ -117,10 +118,13 @@ export default defineComponent({
   <div>
     <Navbar />
     <div class="d-flex">
-      <v-card>
+      <v-card style="height: calc(100vh - 50px)">
```

Comment:
Should we move this style rule behind a CSS class?

Reply:
changed in f932555